pandas/io/formats/style.py

```diff
@@ -1516,13 +1544,13 @@ def _maybe_wrap_formatter(formatter, na_rep: Optional[str]):
     elif callable(formatter):
         formatter_func = formatter
     else:
-        msg = f"Expected a template string or callable, got {formatter} instead"
-        raise TypeError(msg)
+        raise TypeError(
+            f"Expected a template string or callable, got {formatter} instead"
+        )
 
     if na_rep is None:
         return formatter_func
     elif isinstance(na_rep, str):
         return lambda x: na_rep if pd.isna(x) else formatter_func(x)
     else:
-        msg = f"Expected a string, got {na_rep} instead"
-        raise TypeError(msg)
+        raise TypeError(f"Expected a string, got {na_rep} instead")
```

Comment:
for things like this that are fine either way, we'd prefer you leave them alone.  Means less work for reviewers.

Reply:
Got it!